convert-to-declarative-plugin/pom.xml

```diff
@@ -208,12 +204,6 @@
       <version>2.9</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-support</artifactId>
-      <version>3.1</version>
-      <scope>test</scope>
```

Comment:
This is a transitive, compile-scope dependency of workflow-job, so there is no need to depend on it explicitly in test scope. There was a RequireUpperBoundDeps conflict, so I moved it to dependency management in the parent pom, and everything seems to be ok.